featurebyte/worker/task/target_table.py

```diff
@@ -0,0 +1,59 @@
+"""
+TargetTable creation task
+"""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from featurebyte.logging import get_logger
+from featurebyte.models.target_table import TargetTableModel
+from featurebyte.schema.worker.task.target_table import TargetTableTaskPayload
+from featurebyte.service.target_table import TargetTableService
+from featurebyte.worker.task.base import BaseTask
+from featurebyte.worker.task.mixin import DataWarehouseMixin
+
+logger = get_logger(__name__)
+
+
+class TargetTableTask(DataWarehouseMixin, BaseTask):
+    """
+    TargetTableTask creates a TargetTable by computing historical features
+    """
+
+    payload_class = TargetTableTaskPayload
+
+    async def execute(self) -> Any:
+        """
+        Execute TargetTableTask
+        """
+        # TODO: fully implement in follow-up
```

Comment:
will implement in follow-up to try to consolidate w/ historical feature tables as well